### DISCUSSION TITLE
React Native: Prevent / Detect voice recording (Android)

### DISCUSSION THREAD

**Discussion by sherbiiny** [gbumps/react-native-screenguard#54] [Q&A] [answered]
Is there a way to prevent or detect if the user is voice recording in a React Native app?

**Comment by gbumps** [comment, marked as answer]
Hi @sherbiiny, thanks for reaching out.

However, there are no way you can check if phone is recording by another app due to restriction for security reason. An app cannot access information about another app's internal state, including audio recording. This could easily lead to app rejection.
